src/collective/collectionfilter/resources/collectionfilter-bundle-compiled.js

```diff
@@ -234,11 +235,18 @@ define('collectionfilter',[
 
         reloadCollection: function (collectionURL) {
             var cl = new this.contentloader(this.$el, {
-                url: collectionURL,
+                url: collectionURL + '&ajax_load=1',
                 target: '#content-core',
                 content: '#content-core',
                 trigger: 'immediate'
             });
+            // TODO: remove this, once ``contentloader`` handles history
+            // updates itself and adds ajax_load.
+            window.history.replaceState(
```

Comment:
using ``replaceState`` instead of ``updateState`` to not clutter the history with all the filter attempts...